README.md

```diff
@@ -10,14 +10,31 @@ To add a conference to the list, please [contribute](#contributing)!
 |------------|------|-------|-------------|
 | [FunctionalConf](https://functionalconf.com/) | 12-16 Dec | The LaLiT Ashok Hotel, Bengaluru | Bring together the growing community of functional programmers under one roof. |
 | [Devops Days India](http://devopsdaysindia.org/) | 8-9 Dec | Sheraton Grand, Bengaluru | Devopsdays is a worldwide series of technical conferences covering topics of software development, IT infrastructure operations, and the intersection between them. | 
+| [DataHack Summit](https://www.analyticsvidhya.com/datahack-summit-2018/) | 22-24 Nov | NIMHANS Convention Center, Bengaluru | Conference On Artificial Intelligence, Machine Learning, Deep Learning & IOT | 
+| [NASSCOM Product Conclave](http://productconclave.in/) | 25-26 Oct | Vivanta by Taj, Bengaluru | NPC has always been synonymous with the amazing entrepreneurial ecosystem of innovative software products developed, engineered, marketed and sold from India and the globe. NPC, this year will discuss how to creatively enhance businesses when the next generation of technologies, including machine intelligence, is everywhere ! | 
```

Comment:
Please remove this conference since this does not have a CFP. If it has, I'm happy to be pointed to it. :)